catboost/jvm-packages/catboost4j-prediction/pom.xml

```diff
@@ -107,6 +107,7 @@
                     <encoding>
                         utf-8
                     </encoding>
+                    <source>8</source>
```

Comment:
Why have you added this line?

Reply:
https://bugs.java.com/bugdatabase/view_bug.do?bug_id=8212233 -
(maven + javadoc + jdk 11-12) the problem is not solved.